docs/CODEOWNERS

```
@@ -194,3 +194,10 @@
 
 # Ori and the Blind Forest
 # /worlds_disabled/oribf/ <Unmaintained>
+
+###################
+## Documentation ##
+###################
+
+# Apworld Dev Faq
+/docs/apworld dev faq.md @qwint @ScipioWright
```

```suggestion
/docs/apworld\ dev\ faq.md @qwint @ScipioWright
```
I think white space needs escape.

---

my alternative solution i didn't get to yet was just to change them to underscores in the doc and the filename, i think that'd be better especially for readability